### ISSUE TITLE
explaining force_plot when feature=0

### ISSUE THREAD

**Issue by ironv** [shap/shap#1823]
My binary classification model was created using `lightgbm` and 120k observations in the training dataset (1's & 0's ~1:1).   The model has an average precision of ~91% and the features make sense.  The summary plot is shown below.
![image](https://user-images.githubusercontent.com/4298134/108158065-290adf00-70b2-11eb-9b25-926c11a84292.png)

The force plot for an observation with `label=1` and very low score is shown (trying to look at observations which will likely be FNs)
![image](https://user-images.githubusercontent.com/4298134/108158279-99b1fb80-70b2-11eb-979a-6d9ba54a1618.png)

My question is regarding the features which have a value of 0 in the input matrix `X` and yet have a shapley value contribution (features 746, 777, 1587, etc. in the above picture).  [This](https://christophm.github.io/interpretable-ml-book/shap.html) source suggests that it can happen when these features are correlated with other feature(s) (with non-zero value in `X`) which influence the prediction.
1. is there a reference to this in any of the published work on `TreeShap`?  Is there any other explanation for those non-zero shap values?
2. is there a function or any other support in `shap` to identify those features (correlated and in `X`)
3. if not, the way I am thinking of adding insight is - (i) for the above observation, identify features which are non-zero in `X`.  (ii) Compute correlation between all those features and feature 746 (iii) repeat for others.  I am hoping there is a better way.
